NitroxServer/GameLogic/PlayerManager.cs

```diff
@@ -171,6 +222,27 @@ public void PlayerDisconnected(NitroxConnection connection)
             }
         }
 
+        public void FinishProcessingReservation()
+        {
+            initialSyncTimer.Dispose();
+            PlayerCurrentlyJoining = false;
+
+            Log.Info($"Finished processing reservation. Remaining requests: {JoinQueue.Count}");
+
+            if (JoinQueue.Count > 0)
+            {
+                var keyValuePair = JoinQueue.Dequeue();
+                NitroxConnection requestConnection = keyValuePair.Key;
+                MultiplayerSessionReservationRequest reservationRequest = keyValuePair.Value;
+                MultiplayerSessionReservation reservation = ReservePlayerContext(requestConnection,
+                    reservationRequest.PlayerSettings,
+                    reservationRequest.AuthenticationContext,
+                    reservationRequest.CorrelationId);
```

Comment:
A few things.

- I know you used var because of the length of the type but we don't use var if it's not 100% needed.
- To remove the clutter I added a new line between the "variable defining" part and the "data processing"
- Imo it's better if the parameter are align with the others and not just indent by 1 tab.

```suggestion
                KeyValuePair<NitroxConnection, MultiplayerSessionReservationRequest> keyValuePair = JoinQueue.Dequeue();
                NitroxConnection requestConnection = keyValuePair.Key;
                MultiplayerSessionReservationRequest reservationRequest = keyValuePair.Value;

                MultiplayerSessionReservation reservation = ReservePlayerContext(requestConnection,
                                                                                 reservationRequest.PlayerSettings,
                                                                                 reservationRequest.AuthenticationContext,
                                                                                 reservationRequest.CorrelationId);
```